### DISCUSSION TITLE
What's happen with next-intl/link?

### DISCUSSION THREAD

**Discussion by huypham1411** [amannn/next-intl#640] [Q&A] [answered]
Hi, in the previous version, I import Link from next-intl/link since I got a warning told that @deprecated — Is available as import Link from 'next-intl/link' now. Now, after I upgrade the version, I cannot find the Link and got module not found message. Hope someone can help. Appreciate!

**Comment by amannn** [comment, marked as answer]
Hey, I think you're looking for this: https://next-intl-docs.vercel.app/blog/next-intl-3-0#new-navigation-apis-for-the-app-router

Hope this helps!

**Reply by huypham1411** [reply]
thanks for the fast response, I rolled back the previous version (which is 3.0.0-beta.19) to solve the module not found issue, not sure why in this version, everything still works normally but got the module not found on the latest version, I use `import Link from 'next-intl/link';` , I'll try this again and let you know if I still have any issues

**Reply by huypham1411** [reply]
OK, I've some spare times and following the documents and solve this issue, thanks a lot @amannn
